common/reviews/api/rush-lib.api.md

```diff
@@ -178,6 +178,24 @@ export interface _IPnpmOptionsJson extends IPackageManagerOptionsJsonBase {
     strictPeerDependencies?: boolean;
 }
 
+// @public
+export interface IRushConfigurationProjectJson {
+    // (undocumented)
```

Comment:
It'd be useful to document these.

Reply:
Sure. Assumably, ripping the descriptions out of the JSON Schema would be ideal, correct? https://developer.microsoft.com/json-schemas/rush/v5/rush.schema.json